Assets/HoloToolkit/UX/Scripts/BoundingBoxes/BoundingBoxGizmoHandle.cs

```diff
@@ -0,0 +1,398 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using HoloToolkit.Unity.InputModule;
+using UnityEngine;
+
+#if UNITY_WSA && UNITY_2017_2_OR_NEWER
+using UnityEngine.XR.WSA;
+#endif
+
+namespace HoloToolkit.UX.BoundingBoxes
+{
+    public class BoundingBoxGizmoHandle : MonoBehaviour, IInputHandler, ISourceStateHandler
+    {
+        public enum RotationType
```

Comment:
If these enums are intended to be public, please move them outside of the class.